Network/HTTP/Types/QueryLike.hs

```diff
@@ -33,21 +37,38 @@ class QueryKeyLike a where
 class QueryValueLike a where
     toQueryValue :: a -> Maybe B.ByteString
 
+-- | @since 0.7.0
 instance (QueryKeyLike k, QueryValueLike v) => QueryLike [(k, v)] where
     toQuery = map (toQueryKey *** toQueryValue)
 
+-- | @since 0.7.0
 instance (QueryKeyLike k, QueryValueLike v) => QueryLike [Maybe (k, v)] where
     toQuery = toQuery . catMaybes
 
+-- | @since 0.7.0
```

Comment:
Don't need to add `@since` notations to all instances that were introduced when the class was introduced.